blocks/index.js

```diff
@@ -7,6 +7,7 @@ import * as query from 'hpq';
 
 export { query };
 export { default as Editable } from './components/editable';
+export { default as EditableQuote } from './components/wp-quote';
```

Comment:
I think our idea for now, is to use the `Editable` component in all the places we need something to be editable. Typically the Editable component could leverage TinyMCE or something. So in the context of the quote for example, you'd use the `Editable` twice, once for the content and once for the citation. (I think the `quote` block from the `per-block` prototype is using this approach, could be an inspiration https://github.com/WordPress/gutenberg/blob/master/docs/tinymce-per-block/src/blocks/quote-block/form.js#L95-L133)